scripts/wasmd/start.sh

```diff
@@ -37,7 +37,7 @@ echo "wasmd running and logging into $WASMD_LOGFILE"
 # sleep 3 && cat "$WASMD_LOGFILE"
 
 # Use a large timeout because of potentially long image download in `docker run`
-if ! timeout 180 bash -c "until docker inspect -f '{{.State.Running}}' '$CONTAINER_NAME' &> /dev/null; do sleep 0.5; done"; then
+if ! timeout 180 bash -c "until [ \"\$( docker container inspect -f '{{.State.Status}}' \"$CONTAINER_NAME\" 2> /dev/null )\" = \"running\" ]; do sleep 0.5; done"; then
```

Comment:
Before, `docker inspect` returned a failure code as long as the container was not available. Once it existed but was not yet running, `docker inspect` succeeded even when the text output was "false".